src/main/resources/OH-INF/thing/thing-esphome.xml

```diff
@@ -47,6 +50,14 @@
 					https://esphome.io/components/api#configuration-variables</description>
 			</parameter>
 
+			<parameter name="allowActions" type="boolean" required="false" groupName="haActions">
```

Comment:
Aside from this being a new feature, what is the rationale for making this configurable and disabled by default?